pennylane/devices/null_qubit.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Xanadu Quantum Technologies Inc.
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
```

Comment:
Are we doing this across the repo? For me we shouldn't assert copyright on this file from 2018 since it didn't exist then.

Reply:
I think it's fairly scattered.

Reply:
i first copy-pasted from default_qubit.py then updated when I saw it said 2023, will put it back to 2022